wrappers/parking_lot/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+//! This crate provides a Shuttle-compatible implementation and wrapper for [`parking_lot`] in order to make it
+//! more ergonomic to run a codebase under Shuttle.
+//!
+//! [`parking_lot`]: <https://crates.io/crates/parking_lot>
+//!
+//! To use this crate, add something akin to the following to your Cargo.toml:
```

Comment:
Also add the `shuttle` feature flag to your crate

Reply:
? Adding `shuttle` feature flag to your crate is the first thing that's done in the code snippet.

Reply:
We need to update this and add the recommendation that by default, users get the latest version of `parking_lot`, and that if they want to depend on a specific version, they should add a dependency naming it `parking_lot_orig`